synth.metadata

```diff
@@ -39,2689 +39,2662 @@
   ],
   "newFiles": [
     {
-      "path": "tslint.json"
+      "path": "package.json"
     },
     {
-      "path": "codecov.yaml"
+      "path": "webpack.config.js"
     },
     {
-      "path": "tsconfig.json"
+      "path": "linkinator.config.json"
     },
     {
-      "path": "CONTRIBUTING.md"
+      "path": ".eslintrc.yml"
     },
     {
-      "path": "renovate.json"
+      "path": ".prettierrc"
     },
     {
-      "path": ".prettierrc"
+      "path": "codecov.yaml"
     },
     {
-      "path": "linkinator.config.json"
+      "path": ".gitignore"
     },
     {
-      "path": ".clang-format"
+      "path": "package-lock.json"
     },
     {
-      "path": ".repo-metadata.json"
+      "path": "tslint.json"
     },
     {
-      "path": ".readme-partials.yml"
+      "path": "tsconfig.json"
     },
     {
-      "path": "CODE_OF_CONDUCT.md"
+      "path": ".nycrc"
     },
     {
-      "path": ".eslintrc.yml"
+      "path": ".clang-format"
     },
     {
       "path": "synth.metadata"
     },
     {
-      "path": ".nycrc"
+      "path": "CONTRIBUTING.md"
     },
     {
-      "path": "LICENSE"
+      "path": "README.md"
     },
     {
-      "path": "CHANGELOG.md"
+      "path": ".repo-metadata.json"
     },
     {
-      "path": "synth.py"
+      "path": ".jsdoc.js"
     },
     {
-      "path": ".eslintignore"
+      "path": "renovate.json"
     },
     {
-      "path": ".jsdoc.js"
+      "path": "CODE_OF_CONDUCT.md"
     },
     {
-      "path": ".prettierignore"
+      "path": "LICENSE"
     },
     {
-      "path": "webpack.config.js"
+      "path": ".readme-partials.yml"
     },
     {
-      "path": "package.json"
+      "path": ".eslintignore"
     },
     {
-      "path": "README.md"
+      "path": ".prettierignore"
     },
     {
-      "path": ".gitignore"
+      "path": "CHANGELOG.md"
     },
     {
-      "path": "package-lock.json"
+      "path": "synth.py"
     },
     {
-      "path": ".git/HEAD"
+      "path": "protos/protos.json"
     },
     {
-      "path": ".git/description"
+      "path": "protos/protos.d.ts"
     },
     {
-      "path": ".git/shallow"
+      "path": "protos/protos.js"
     },
     {
-      "path": ".git/config"
+      "path": "protos/google/cloud/common_resources.proto"
     },
     {
-      "path": ".git/index"
+      "path": "protos/google/cloud/translate/v3beta1/translation_service.proto"
     },
     {
-      "path": ".git/packed-refs"
+      "path": "protos/google/cloud/translate/v3/translation_service.proto"
     },
     {
-      "path": ".git/logs/HEAD"
+      "path": "node_modules/meow/package.json"
     },
     {
-      "path": ".git/logs/refs/remotes/origin/HEAD"
+      "path": "node_modules/meow/node_modules/read-pkg-up/package.json"
     },
     {
-      "path": ".git/logs/refs/heads/autosynth"
```

Comment:
@bcoe I'm not really sure what this data is used for, but I am a little surprised to see both `.git` and `node_modules` here. Seems like if we could sort by path (is that bad?) we might have better diffs, assuming that is something we care about.